content/bash.rst

```diff
@@ -1,7 +1,7 @@
 Bash
 ####
 
-:menu_order: 155
+:menu_order: 015
```

Comment:
ok. thanks... then it might be my fault for not comparing the values the right way.